composer.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "candasm/yii1-mobile-detect-component",
+  "version": "1.0.1",
+  "type": "yii-extension",
+  "description": "MobileDetect library for YiiFramework 1.x",
+  "authors": [
+    {
+      "name": "Candasm",
+      "email": "candas@minareci.net"
+    }
+  ],
+  "config": {
+    "vendor-dir": "protected/vendor/"
+  },
+  "autoload": {
+    "psr-4": {
+      "": "src/"
```

Comment:
[Example](https://github.com/yiisoft/yii/blob/master/framework/collections/CAttributeCollection.php
) class of Yii 1.x framework It does not use namespaces so we don't need to set a namespace. But if you want it for YiiFramework 2 this should be in another version.